modules/light-clients/07-tendermint/light_client_module.go

```diff
@@ -13,17 +13,22 @@ import (
 
 var _ exported.LightClientModule = (*LightClientModule)(nil)
 
+// LightClientModule implements the core IBC api.LightClientModule interface.
```

Comment:
I guess `api` isn't a module yet but I'm happy to keep it here regardless

Reply:
Yeah, I am assuming that we will have it by the time we release this. :)